### PR TITLE
Journalist results tab

### LINKED ISSUE #173: Remove navbar, just have logo (or name) by search bar

### PR DESCRIPTION
**Pull request by VasilyShelkov**
@Winwardo Ready for comments and merge

closes #173

### CODE REVIEW

File: public/css/customFonts.css [hunk 1]
```diff
@@ -80,3 +81,31 @@ a:hover .tweet.fullname {
...
+
+.ui.menu a.item.active:hover{
+  background:#a333c8;
+  color: white
+}
+
+/*.ui.link.menu .item:active,.ui.menu .link.item:active,.ui.menu a.item:active{
+  background:rgba(0,0,0,.03);
+  color:rgba(0,0,0,.95)
+}*/
```
Comment (Winwardo) [inline public/css/customFonts.css]: Don't leave in commented code pls

File: src/client/results/Results.js [hunk 2]
```diff
@@ -1,128 +1,61 @@
...
 const mapStateToProps = (state) => ({
+  searchTerms: state.searchTerms,
   feed: state.feed,
   mostFrequent: state.mostFrequent,
+  showJournalismInfo: state.showJournalismInfo,
+});
+
+const mapDispatchToProps = (dispatch) => ({
+  onClickSocialWebResults: () => dispatch(changeResultsView(false)),
+  onClickJournalismInfo: () => dispatch(changeResultsView(true)),
```
Comment (Winwardo) [inline src/client/results/Results.js]: eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeehhhh, I don't like this being a true/false toggle, even though it's one of two states.

I'd ~prefer~ if this was implemented using some pseudo-enum, e.g. a const TAB_TWEETS = 0; const TAB_JOURNALIST = 1 and that was passed through

For ease of getting this through I won't require this though

File: src/client/results/socialweb/mostfrequent/SearchBar.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import React, { Component } from 'react';
-import { throttleFunction } from '../../../shared/utilities';
+import React from 'react';
+import { throttleFunction } from '../../../../shared/utilities';
```
Comment (Winwardo) [inline src/client/results/socialweb/mostfrequent/SearchBar.js:2]: So many ../ haha

File: src/client/results/Results.js (hunk 2 above)
Comment (VasilyShelkov) [inline src/client/results/Results.js]: yer was thinking that, then was like, well there's never gonna be more than two tabs so seems a waste X)

Comment (VasilyShelkov) [inline src/client/results/Results.js]: happy to change it if you'd prefer

Comment (Winwardo) [inline src/client/results/Results.js]: If it's not a hassle

**Comment by VasilyShelkov** [comment]
@Winwardo fixed comments, check that you're happy with the tab index over boolean method :) 

Seriously should start getting up earlier haha, most of the day has already disappeared!

**Pull request merged**